chcemvediet/templates/main/homepage/casestudy/base.html

```diff
@@ -0,0 +1,16 @@
+{# vim: set filetype=htmldjango shiftwidth=2 :#}
+
+{% comment %}
+ %
+ % Context:
+ %  -- None
+ %
+{% endcomment %}
+
+{% block case_study %}
```

Comment:
Tu ti ostalo `case_study`. Zjednot to rovnako ako vsade.

Reply:
:ok: